exercises/accumulate/examples/src/Accumulate.purs

```diff
@@ -0,0 +1,14 @@
+module Accumulate
+  ( accumulate
+  ) where
+
+import Data.List (List(Nil), (:), reverse)
+
+
+accumulate :: forall a b . (a -> b) -> List a -> List b
+accumulate f values = accumulate' f Nil values
```

Comment:
I just find TCO easier to read and write these days. I don't think that exercism actually exposes these example solutions to students anywhere so I'm not really sure what we're aiming for here. Should it be trying to be beginner friendly, or should it try to be a more robust solution?